packages/examples/packages/ethereum-provider/src/index.ts

```diff
@@ -88,6 +118,20 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
     case 'getAccounts':
       return await getAccounts();
 
+    case 'personalSign': {
+      const accounts = await getAccounts();
+      assert(request.params, 'Must pass parameters');
+      assert(
+        hasProperty(request.params, 'message'),
+        'Must pass parameters containing a message to sign',
+      );
+      assert(
+        typeof request.params.message === 'string',
+        'Must pass message to sign as a string',
+      );
```

Comment:
We don't really do validation in any of the examples. I think it's fine to just do a type cast for now.